autoblocks/_impl/testing/run.py

```diff
@@ -300,8 +243,16 @@ def run_test_suite(
     # How many evaluators to run concurrently on the result of a test case
     max_evaluator_concurrency: int = 5,
 ):
-    init_global_vars_and_event_loop()
+    cli_server_address = AutoblocksEnvVar.CLI_SERVER_ADDRESS.get()
+    if not cli_server_address:
+        raise RuntimeError(
+            "Autoblocks tests must be run within the context of the testing CLI.\n"
+            "Make sure you are running your test command with:\n"
+            "$ npx autoblocks testing exec -- <your test command>"
+        )
 
+    global_state.init()
+    global_state.http_client().base_url = cli_server_address
```

Comment:
this is a package-wide global, you can't modify it like this. it would affect users consumers of the client

Reply:
Ah yeah your right. Maybe client doesn't need to exist here at all - its pretty light and conceptually doesn't relate much to the event_loop

Reply:
we still want a global http client, especially for the pieces used in production like the tracer and prompt sdk

https://www.python-httpx.org/advanced/clients/

also it doesn't need to relate to the event loop. this file is just to maintain package-wide globals

Reply:
i am saying for testing in particular you can either refactor so that you can use the package-wide http client or use your own client (still a global, but isolated to testing) like it was set up previously

Reply:
Ended up going to the latter option - still have client in `global_state` but added a different client to the testing